src/components/Notice/Notice.jsx

```diff
@@ -0,0 +1,77 @@
+/* eslint react/no-array-index-key: 0 */
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { Alert, Button, Container } from 'react-bootstrap'
+
+const Notice = ({ alert, buttonProps, dismissible, withBackButton }) => {
+  const [show, setShow] = useState(true)
+  const { title, body, variant } = alert
+  let onClick
+  let text
+  if (withBackButton) ({ onClick, text } = buttonProps)
+
+  return (
+    show && (
+      <Container>
+        <Alert className='my-5 text-break' variant={variant} onClose={() => setShow(false)} dismissible={dismissible}>
+          {title && (
+            <Alert.Heading>{title}</Alert.Heading>
+          )}
+          {body.map((message, index) => {
+            // If "message" is a JSON string, return it as a formatted code block
+            // Otherwise, return it as a regular string
+            const isJSONParsable = () => {
+              try {
+                JSON.parse(message)
+                return true
+              } catch (error) {
+                return false
+              }
+            }
+            const isJavascriptObject = isJSONParsable(message)
+
+            return (
+              isJavascriptObject ? (
+                <pre className='mb-0' key={index}>{message}</pre>
+              ) : (
+                <p className='mb-0' key={index}>{message}</p>
+              )
```

Comment:
a basic string would still render in the `<pre>` tag, but it would change the font. when updating client repo's to use their own typeface, I wanted it to be able to apply here too. (code block errors are only see in dev)